src/lib/swapper/swappers/ThorchainSwapper/getThorTradeQuote/getTradeQuote.ts

```diff
@@ -171,25 +171,29 @@ export const getThorTradeQuote = async (
   const streamingSwapQuote = maybeStreamingSwapQuote?.unwrap()
 
   // recommended_min_amount_in should be the same value for both types of swaps
-  const recommendedMinAmountInCryptoBaseUnit = swapQuote.recommended_min_amount_in
-    ? convertPrecision({
-        value: swapQuote.recommended_min_amount_in,
-        inputExponent: THORCHAIN_FIXED_PRECISION,
-        outputExponent: sellAsset.precision,
-      })
-    : undefined
-
-  if (
-    recommendedMinAmountInCryptoBaseUnit &&
-    bn(sellAmountCryptoBaseUnit).lt(recommendedMinAmountInCryptoBaseUnit)
-  ) {
-    return Err(
-      createTradeAmountTooSmallErr({
-        minAmountCryptoBaseUnit: recommendedMinAmountInCryptoBaseUnit.toFixed(),
-        assetId: sellAsset.assetId,
-      }),
-    )
-  }
+  // TODO(gomes): We may or may not want to bring this back in at some point, but the recommended_min_amount_in is currently way too high,
```

Comment:
Happy to remove instead of commenting if we prefer to - we can always circle back to this commit and revert it

Reply:
>The recommended minimum inbound amount for this transaction type & inbound asset. Sending less than this amount could result in failed refunds.

I think we should keep this logic and just add a warning for the user so they know the risk. Then we are covered across the board.

Reply:
alternative per dms would be making our own min logic which wouldn't be too complicated that has slightly lower buffers, but still something "reasonable". (ie. thorchain is using 4x outbound fee buffer, where we could do 2x and maybe a warning?)

Reply:
agree a ui warning would be handy, e.g "thorchain recommends at least xyz ..."